src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,66 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline.
+ *
+ * @author Pearlynn
+ */
+
+public class Deadline extends Task {
+
+    /**
+     * The deadline of the task.
+     */
```

Comment:
Not sure if this is required, but:

Javadoc of class members can be specified on a single line as follows:

```suggestion
    /** The deadline of the task. */
```